src/Serilog.Sinks.Elasticsearch/Sinks/ElasticSearch/ElasticsearchSinkOptions.cs

```diff
@@ -281,6 +281,11 @@ public int QueueSizeLimit
         /// </summary>
         public RollingInterval BufferFileRollingInterval { get; set; }
 
+        /// <summary>
+        /// Apikey to access elasticsearch, the value is encoded, atob(`id:apikey`).This is another method instead user/password
```

Comment:
Can we make the comment a bit more descriptive? As I understood, the API key needs to be a concatenation of the id, a ':' and the apikey and then base64 encoded? I m not sure everybody understands the atob